aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/BaseHandlerStd.java

```diff
@@ -280,44 +286,73 @@ private ProgressEvent<ResourceModel, CallbackContext> modifyParameters(final Pro
-        Map<String, Parameter> parametersToModify = getParametersToModify(model.getParameters(), currentDBParameters);
-        for (List<Parameter> paramsPartition : Iterables.partition(parametersToModify.values(), MAX_PARAMETERS_PER_REQUEST)) {  //modify api call is limited to 20 parameter per request
-            ProgressEvent<ResourceModel, CallbackContext> progressEvent = modifyParameters(proxyClient, proxy, callbackContext, paramsPartition, model);
-            if (progressEvent.isFailed()) return progressEvent;
+        final ResourceModel model = progress.getResourceModel();
+        final CallbackContext context = progress.getCallbackContext();
+        final Map<String, Parameter> parametersToModify = getParametersToModify(model.getParameters(), currentDBParameters);
+
+        try {
+            for (final List<Parameter> partition : Iterables.partition(parametersToModify.values(), MAX_PARAMETERS_PER_REQUEST)) {
+                proxyClient.injectCredentialsAndInvokeV2(
+                        Translator.modifyDbClusterParameterGroupRequest(model, partition),
+                        proxyClient.client()::modifyDBClusterParameterGroup
+                );
+            }
+        } catch (Exception exception) {
+            return Commons.handleException(ProgressEvent.progress(model, context),
+                    exception,
+                    DEFAULT_DB_CLUSTER_PARAMETER_GROUP_ERROR_RULE_SET);
         }
-        return ProgressEvent.progress(model, callbackContext);
+
+        return ProgressEvent.progress(model, context);
     }
 
-    private ProgressEvent<ResourceModel, CallbackContext> modifyParameters(final ProxyClient<RdsClient> proxyClient,
-                                                                           final AmazonWebServicesClientProxy proxy,
-                                                                           final CallbackContext callbackContext,
-                                                                           final List<Parameter> paramsPartition,
-                                                                           final ResourceModel model) {
-        return proxy.initiate("rds::modify-db-cluster-parameter-group", proxyClient, model, callbackContext)
-                .translateToServiceRequest((resourceModel) -> Translator.modifyDbClusterParameterGroupRequest(resourceModel, paramsPartition))
-                .makeServiceCall((request, proxyInvocation) -> proxyInvocation.injectCredentialsAndInvokeV2(request, proxyInvocation.client()::modifyDBClusterParameterGroup))
-                .handleError((describeDbParameterGroupsRequest, exception, client, resourceModel, ctx) ->
-                        Commons.handleException(
-                                ProgressEvent.progress(resourceModel, ctx),
-                                exception,
-                                DEFAULT_DB_CLUSTER_PARAMETER_GROUP_ERROR_RULE_SET))
-                .progress();
```

Comment:
Reason for change: no benefits from using the call chain and a potential flooding of the CallbackContext object with a massive `DescribeDBClusters` response

Reply:
I think the parameter was misnamed. This was a modify response. 

The benefit of saving the call to callback context is logging. There are ways to collect those logs elsewhere, but I think we should consider logging the response to our request logger directly if we do not add the response to callback context.

Reply:
The request/response logging is facilitated by the [LoggingProxyClient](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds/blob/master/aws-rds-cfn-common/src/main/java/software/amazon/rds/common/logging/LoggingProxyClient.java#L70-L75), hence i expect no changes here in fact.